src/containers/ReviewModal/ReviewModal.scss

```diff
@@ -8,6 +8,7 @@
 
   .review-actions-username {
     flex-grow: 1;
+    padding: map-get($spacers, 1);
```

Comment:
This should be `padding-bottom: ($spacers, 3)` (16px in design), and should be  in the media breakpoint config.
As is, the padding is too small and is also there when in the wide view.